### PR TITLE
Tabbed navigation for wp.com plans and signup

### PR DESCRIPTION
**Pull request by adamziel**
This PR updates plans page both on /plans and in signup to have personal/business tabs.

Related to the blogger plan project p9jf6J-Sn-p2

**Testing instructions:**

1. Follow instructions in D20180-code and make sure you are in the `test` group
1. Confirm signup steps with plan selection looks like this:

zrzut ekranu 2018-10-29 o 15 58 08
zrzut ekranu 2018-10-29 o 15 58 12

1. Confirm plans page looks like this:

zrzut ekranu 2018-10-29 o 15 58 02
zrzut ekranu 2018-10-29 o 15 57 24

1. Follow instructions in D20180-code and make sure you are in the `control` group
1. Confirm that you can't any see tabs on `/plans` or during the signup
1. Confirm that jetpack signup flow works just like in production

### CODE REVIEW

**Comment by jancavan** [comment]
As mentioned in Slack, it does end up looking really cramped with 4-columns, even at ~1200px.

There's two options I thought of for viewports between 960 - 1200:

### 1) Arrange the plans in a grid:
pricing-grid

This is a really quick mockup so please ignore the spacing/alignment issues :)

### 2. Arrange them in a row similar to:

screenshot 2018-09-19 13 49 59

I think this is also the plan for our new signup flow with more personality. I'll follow up on that to make sure.

One thing I'm not too keen about this approach is we'll probably need to create a non-standard breakpoint. However, I think we already use one which is 750px, so anything between that and 960px can use the current stacked layout.

@adamziel Another update we need to do are the icons. They should match the ones on the current `/pricing` page.

**Comment by anneforbush** [comment]
@jancavan I like Option 2 (seems easier to scan and read), though we wouldn't want to re-use the exact same styling and look-and-feel from the new "personality" flow, I imagine. So basically, I'm seeing the layout of Option 2 but with Calypso styling -- is that how you're envisioning it?

**Comment by jancavan** [comment]
@anneforbush Yup, that's exactly it. We still want to maintain Calypso styles.

**Comment by dezzie** [comment]
Putting an abridged version of what I mentioned in Slack:
- I don't think we should get rid of columns, it makes it much harder for customers to comparison shop between the plans.
- It also is going to make it trickier for Marketing and HE to explain the nuances between the Blogger plan and the rest of the plans. The Premium and Business plans also are in the process of being re-worked. 
- A stacked approach relieves the cramped copy situation, but then there's no way to compare the different features on our plans, or between our competitors and our plans as well.

Playing off of what @jancavan proposed in the first 2x2 idea, the first row with the Blogger and Personal plan are essentially both versions of Personal-use sites. The second row has the business ones. But if you did something like this:

blogger-plan_1
- you can compare the added value between Free and Blogger
- and if we end up with more plan options we can always kick out the Free plan from the 3rd column

**Comment by dezzie** [comment]
An alternative to the toggle option is to have 2 rows of 3 columns. This is similar to the 2x2 idea above, except it allows a comparison with the Free plan. It also keeps focus on the plans that are relevant to the customer. If I'm only interested in blogging, I can safely ignore everything under "Business Plans"
blogger-plan_3

**Comment by jancavan** [comment]
Those are interesting ideas @dezzie! Between the two options, I think I like the second one better. My only concern is Premium/Business get bumped down the page and people might miss it. With the first one, I'm also worried they're hidden behind a tab. I agree that hiding Free is a bit of a dark pattern, but I'm not sure if it's ok to surface that in the expense of Premium/Business.

I think grouping the plans like this is going to require a bigger discussion with Marketing though. From my understanding, this is going to be a 50/50 split test, so it might also be tricky to determine what worked and what didn't since this is a completely different layout to be tested against the Control. That said, I think these are all interesting ideas we can explore if this test performs well.

**Comment by jancavan** [comment]
Looking good so far @adamziel! I did a quick review and here are a couple things I noticed:

1) The popover positioning normally adjusts pretty well to the screen size as you scale down the browser, but in this PR, I had to refresh to get it to position correctly:

screenshot 2018-10-29 10 48 23

2) The text wraps around the gridicon at narrow widths:

screenshot 2018-10-29 10 54 10

We can just remove `margin-left: 10px;` from `plan-features__item-title` and  add `margin-right: 10px;` to `gridicons-checkmark` like so:

screenshot 2018-10-29 10 55 51

3) I think it's better to align the info icon top instead of middle. We can just remove `align-self: center;` from `plan-features__item-tip-info` entirely:

screenshot 2018-10-29 10 58 55

4) This is an already existing issue that may need to just be revisited, but it doesn't do very well with accessibility; none of the items within the table are focusable.

File: client/my-sites/plan-features/style.scss [hunk 1]
```diff
@@ -486,6 +519,13 @@ $plan-features-sidebar-width: 272px;
 	overflow-x: auto;
 }
 
+.signup {
+	.is-customer-type-toggle {
```
Comment (jancavan) [inline client/my-sites/plan-features/style.scss]: No need to nest. Can just be `.signup .is-customer-type-toggle`

File: client/my-sites/plan-features/style.scss [hunk 2]
```diff
@@ -305,6 +312,33 @@ $plan-features-sidebar-width: 272px;
...
+		padding-top: 8px;
+	}
+
+	.segmented-control__item {
+		min-width: 50%;
+	}
+
+    @include breakpoint( '<480px' ) {
+		margin-bottom: 9px;
+		.segmented-control__item {
```
Comment (jancavan) [inline client/my-sites/plan-features/style.scss]: Small nitpick: needs space above `.segmented-control__item`

File: client/my-sites/plan-features/style.scss [hunk 3]
```diff
@@ -69,6 +69,13 @@ $plan-features-sidebar-width: 272px;
 	height: 50px;
 }
 
+.plan-features__table.has-2-cols {
+	display: block;
+	max-width: 720px;
+	margin-left: auto;
```
Comment (jancavan) [inline client/my-sites/plan-features/style.scss]: I think this can just be `margin: 0 auto`?

Comment (jancavan) [reply client/my-sites/plan-features/style.scss]: Instead of adding `margin-left` and `margin-right`.

**Comment by jancavan** [comment]
When you aren't the primary site owner, there is a notice that appears between the tabs and table. It also just appears in one of the views instead of both. We can either move it above the tabs or have it behave like a Global Notice: https://wpcalypso.wordpress.com/devdocs/design/global-notices

The messaging is also unclear. Maybe reword it to: "Plan subscriptions can only be managed by site owners"? cc @sararosso 

screenshot 2018-10-29 11 58 02

File: client/my-sites/plan-features/style.scss (hunk 3 above)
Comment (adamziel) [reply client/my-sites/plan-features/style.scss]: margin-top is set by another rule and we want to set just left and right to auto

**Comment by sararosso** [comment]
Just pulling out the request - 

Original
> You need to be the plan owner to manage this plan. 

Suggestion
> Maybe reword it to: "Plan subscriptions can only be managed by site owners"?

I think the reasoning behind the notice isn't what you mentioned @jancavan - the site owner does not have to be the plan owner, as sometimes another user on the site will actually purchase the upgrade for the site. You *do* need to be the plan owner to manage changes to the plan, so I believe the current wording is appropriate / correct.

**Comment by jancavan** [comment]
@sararosso Ah, makes sense, thanks for the clarification.

The original is:
> You need to be the plan owner to manage this **site**.

Should it be changed to:
> You need to be the plan owner to manage this **plan**.

**Comment by sararosso** [comment]
> Should it be changed to:
> You need to be the plan owner to manage this plan.

I believe this would be the best wording for this, yes.

**Comment by adamziel** [comment]
I'd update the wording in another PR so we don't have to wait for translations here

File: public/images/plans/plan-blogger.svg [hunk 4]
```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jancavan) [inline public/images/plans/plan-blogger.svg]: I think this should be minified too?

Comment (adamziel) [reply public/images/plans/plan-blogger.svg]: It should! This is just a placeholder though, I only just received the final icon

**Comment by niranjan-uma-shankar** [comment]
I kind of liked the compare interface on apple.com, and was wondering if something on those lines would fit our purposes - showing the plans on a dropdown, instead of a tabbed approach. Not saying one is better than the other(I like the tabbed approach), just sharing it as an alternative :). The advantage I see with the dropdown approach is that I can choose to compare a `Personal` plan with a `Premium` plan, which might not be possible with a tabbed approach if they end up being on different tabs.

screenshot 2018-11-03 at 10 38 48 am

**Comment by sararosso** [comment]
@niranjan-uma-shankar probably best for a p2 - there's a thread here on the latest design: p9jf6J-14t
